sklearn/ensemble/tests/test_partial_dependence.py

```diff
@@ -265,30 +272,22 @@ def test_plot_partial_dependence_multiclass():
                   grid_resolution=grid_resolution)
 
 
-def test_warning_raised_partial_dependence():
-    # Test that deprecation warning is raised
-
-    clf = GradientBoostingRegressor(n_estimators=10, random_state=1)
-    clf.fit(boston.data, boston.target)
-    grid_resolution = 25
-
-    assert_warns_message(DeprecationWarning, "The function "
-                         "ensemble.partial_dependence has been deprecated ",
-                         partial_dependence, clf, [0], X=boston.data,
-                         grid_resolution=grid_resolution)
-
-
-@if_matplotlib
-def test_warning_raised_partial_dependence_plot():
-    # Test that deprecation warning is raised
-
+@pytest.mark.parametrize(
```

Comment:
This parametrization doesn't help readability IMHO, and I still don't see why you would want to change these tests. They will all be removed in 0.23.
But ok...